language-server/modules/langserver-core/src/test/resources/inlayhint/source/inlayhint3.bal

```diff
@@ -0,0 +1,10 @@
+public client class Client {
+    resource function get hello(string name) returns string {
+        return "Hello" + name;
+    }
+}
+
+public function test() {
+    Client cl = new();
+    string result  = cl -> /hello("Ballerina");
```

Comment:
Shall we add tests cases with rest-args?

Reply:
Done